src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -283,6 +298,8 @@ namespace OpenMS
     /// Assignment operator (not implemented)
     SpectrumMetaDataLookup& operator=(const SpectrumMetaDataLookup&);
 
+    /// adds Ion matches to identifcations
+    static void addIonMatches_(PeptideIdentification& pi, const MSSpectrum<Peak1D> &spec);
```

Comment:
:wink: yeah in a perfect world, maybe. No, MSGF+ is the **_only_** one to report such thing.
It clearly is lookup as the spectra have to be looked up. What the class is doing.